### DISCUSSION TITLE
Description required

### DISCUSSION THREAD

**Discussion by jkramps** [gwleuverink/clickup-time-tracker#11] [Ideas]
I believe the description should be required. When we generate time tracked dashboard this will be self-explanatory towards the client. Maybe create a remark that the description is visible for clients.

**Comment by gwleuverink** [comment]
I agree. Proper validation still has to  be done. I'll keep this in mind

**Reply by jkramps** [reply]
Volgens mij is deze done. Misschien de minimum chars naar 3? Nu kun je niet bijv gewoon 'Lunch' gebruiken

**Reply by jkramps** [reply]
En dan nog het wijzigen van een omschrijving

**Comment by gwleuverink** [comment]
done in next release
